### ISSUE TITLE
spring-boot-gradle-plugin: provide sensible defaults for embeddedLaunchScriptProperties

### ISSUE THREAD

**Issue by oneiros-de** [spring-projects/spring-boot#4458]
The gradle plugin sets the values of `initInfoProvides`, `initInfoShortDescription` and `initInfoDescription` to default values that are not taken from the project (e.g. "spring-boot-application"), while the maven plugin uses values from the project (e.g. `${project.artifactId}`).

The gradle plugin should also use values from the project.

**Comment by philwebb** [comment]
Are there any common properties that we can use for Gradle builds? I think the main reason we left it out was there wasn't an obvious candidate.

Perhaps we can use `jar.baseName`?

**Comment by wilkinsona** [comment]
Yes, it's still applicable. Let's look at in 2.1 and either implement something or close it.
